### ISSUE TITLE
Error in backend module "News Admin"

### ISSUE THREAD

**Issue by yurigargamel** [georgringer/news#604]
This might be connected to issue #316.

When trying to access the news sysfolder via back end module "news", I get this error:

> Oops, an error occurred!
> Argument 1 passed to TYPO3\CMS\Fluid\ViewHelpers\Be\Widget\PaginateViewHelper::render() must implement interface TYPO3\CMS\Extbase\Persistence\QueryResultInterface, string given

I already tried clearing all custom templates like this:
`plugin.tx_news.view.partialRootPaths.1 >`
`plugin.tx_news.view.templateRootPaths.1 >`
`plugin.tx_news.view.layoutRootPaths.1 >`

But this didn't change anything (these are for frontend rendering anyways).

What could be the problem?

TYPO3 8.7.8
tx_news 6.3.0

**Comment by yurigargamel** [comment]
Ok, we solved it, but don't understand it completely.

Problem was that we defined partialRootPath, templateRootPath and layoutRootPath via constants to point to directories within a custom extension. The custom extension contained all the directories and sub directories from EXT:news/Resources/Private. But all these directories and files were taken from tx_news 5.x and the current setup was a tx_news 6.3.0. That's what caused the error. We solved it by copying all the Resources/Private directories and files from tx_news 6.3.0 to our custom extension.

What we don't understand is why the clearing of the TS values (see initial post) didn't help us get rid of the error message.
